src/User32/User32+DIALOG_DPI_CHANGE_BEHAVIORS.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains nested type <see cref="DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS"/>
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// In Per Monitor v2 contexts, dialogs will automatically respond to DPI changes by resizing themselves and re-computing the positions of their child windows (here referred to as re-layouting).
+        /// This enum works in conjunction with SetDialogDpiChangeBehavior in order to override the default DPI scaling behavior for dialogs.
+        /// This does not affect DPI scaling behavior for the child windows of dialogs(beyond re-layouting), which is controlled by <see cref="DIALOG_CONTROL_DPI_CHANGE_BEHAVIORS"/>.
+        /// </summary>
+        public enum DIALOG_DPI_CHANGE_BEHAVIORS : int
```

Comment:
At least some uses of this enum treat this as a flags enum (where multiple values can be set). Can you add `[Flags]` here?